dali/test/python/test_operator_jpeg_compression_distortion.py

```diff
@@ -72,11 +70,13 @@ def _compare_to_cv_distortion(in_img, out_img, q, no):
     decoded_img = cv2.cvtColor(decoded_img_bgr, cv2.COLOR_BGR2RGB)
 
     diff = cv2.absdiff(out_img, decoded_img)
-    diff_in_range = np.average(diff) < 5, f"Absolute difference with the reference is too big: {np.average(diff)}"
+    diff_in_range = np.average(
+        diff) < 5, f"Absolute difference with the reference is too big: {np.average(diff)}"
```

Comment:
Isn't this piece of code broken?

`diff_in_range` is a pair, thus it is always true-ish and the assert never fails and does not actually enforce the `np.average(diff) < 5` condition.